Octokit.Tests/Clients/RepositoryPagesClientTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿using NSubstitute;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Octokit.Tests.Clients
+{
+    public class RepositoryPagesClientTests
+    {
+        public class TheGetMethod
+        {
+            [Fact]
+            public void RequestsCorrectUrl()
+            {
+                var connection = Substitute.For<IApiConnection>();
+                var client = new RepositoryPagesClient(connection);
+
+                client.Get("fake", "repo");
+
+                connection.Received().Get<Page>(Arg.Is<Uri>(u => u.ToString() == "repos/fake/repo/pages"), null);
```

Comment:
Not sure how these tests should have passed before the merge, as I managed to break an old PR when I merged `master` back in. See 5ca6633cb133ca8db4f2be17c2ebb0e26a446d64 for the fix to these tests.